src/bundler/bundle_v2.zig

```diff
@@ -1973,6 +1982,7 @@ pub const BundleV2 = struct {
     }
 
     pub fn deinit(this: *BundleV2) void {
+        // this.deferred_tasks.deinit(bun.default_allocator);
```

Comment:
Did you mean to leave this commented out?

Reply:
yep the field doesn't exist anymore, removed the comment